app/Actions/Service/RefreshServices.php

```
@@ -0,0 +1,41 @@
+
+namespace App\Actions\Service;
+
+use App\Jobs\Service\RefreshServicesJob;
+use App\Models\Server;
+use Illuminate\Support\Facades\Cache;
+
+class RefreshServices
+{
+    public function refresh(Server $server): bool
+    {
+        if (! Cache::add(self::flagKey($server), true, self::flagTtl($server))) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Cache' in method 'refresh'. (undefined)

(StaticAccess)